format/src/main/scala/scala/meta/internal/prettyprinters/TreePrinter.scala

```diff
@@ -476,55 +503,88 @@ class TreePrinter private ()(implicit val trivia: AssociatedTrivias)
           print(t.name),
           t.tparams,
           Nil,
+          Nil,
           None,
           dBody(t.body)
         )
       case t: Defn.Val =>
-        dDefPats(t.mods, `val`, t.pats, Nil, Nil, t.decltpe, dBody(t.rhs))
+        dDefPats(
+          t.mods,
+          `val`,
+          t.pats,
+          Nil,
+          Nil,
+          Nil,
+          t.decltpe,
+          dBody(t.rhs, t.`=`)
+        )
       case t: Defn.Var =>
         val drhs = Some(t.rhs.fold(Term.Placeholder(): Term)(identity))
-        dDefPats(t.mods, `var`, t.pats, Nil, Nil, t.decltpe, dBodyO(drhs))
+        dDefPats(t.mods, `var`, t.pats, Nil, Nil, Nil, t.decltpe, dBodyO(drhs))
       case t: Decl.Val =>
-        dDefPats(t.mods, `val`, t.pats, Nil, Nil, Some(t.decltpe), empty)
+        dDefPats(t.mods, `val`, t.pats, Nil, Nil, Nil, Some(t.decltpe), empty)
       case t: Decl.Var =>
-        dDefPats(t.mods, `var`, t.pats, Nil, Nil, Some(t.decltpe), empty)
+        dDefPats(t.mods, `var`, t.pats, Nil, Nil, Nil, Some(t.decltpe), empty)
       case t: Decl.Type =>
-        dDef(t.mods, `type`, print(t.name), t.tparams, Nil, None) +
+        dDef(t.mods, `type`, print(t.name), t.tparams, Nil, Nil, None) +
           print(t.bounds)
       case t: Defn.Class =>
         val dctormods =
           if (t.ctor.mods.isEmpty) empty else space + dMods(t.ctor.mods) + space
         val dsignature =
-          dApplyBracket(print(t.name), t.tparams) +
+          dApplyBracket(print(t.name), t.tparams, t.tparamsSeparator) +
             dctormods +
-            dParamss(t.ctor.paramss)
+            dParamss(t.ctor.paramss, t.ctor.paramsSeparator)
         spaceSeparated(
           dMods(t.mods) ::
             `class` ::
             dsignature ::
             Nil
         ) + print(t.templ)
       case t: Pkg.Object =>
-        val ddefn =
-          dDef(t.mods, `package` + space + `object`, print(t.name), Nil, Nil)
-        ddefn + print(t.templ)
+        spaceSeparated(
+          List(
+            dMods(t.mods),
+            `package` + space + `object`,
```

Comment:
what happens with `package /* BOOM */ object`? 😄 

People deserve to loose those comments 😈

Reply:
That's the remaining 1%.